Formula/libtoxcore@0.1.rb

```diff
@@ -0,0 +1,55 @@
+class LibtoxcoreAT01 < Formula
+  desc "Tox library written in C"
+  homepage "https://tox.chat"
+  head "git://github.com/TokTok/c-toxcore", branch: "0.1.x"
```

Comment:
Use the v0.1.11 tag instead. That branch was never released and should be deleted.

Reply:
Thanks mate!